app/helpers/comments_helper.rb

```diff
@@ -108,7 +108,8 @@ def can_reply_to_comment?(comment)
       comment_parent_hidden?(comment) ||
       blocked_by_comment?(comment) ||
       blocked_by?(comment.ultimate_parent) ||
-      guest? && admin_settings.guest_comments_off?)
+      (guest? && admin_settings.guest_comments_off?) ||
```

Comment:
Since the cases are getting lengthy, should we refactor this to use guard clauses?

Reply:
I don't think so, it‘s still very readable to me.

Reply:
I agree with weeklies, can we refactor